### PR TITLE
Update Kermit logging

### PR DESCRIPTION
**Pull request by bjdupuis**
Updating the Kermit version. Allows a greatly simplified usage.

### CODE REVIEW

File: transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.genesys.cloud.messenger.transport.util.logs
+
+import okhttp3.logging.HttpLoggingInterceptor
+
+internal fun Log.okHttpLogger(): HttpLoggingInterceptor.Logger =
+        HttpLoggingInterceptor.Logger { message -> kermit.i(message) }
```
Comment (crumpf) [inline transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt:6]: The linter is complaining about spacing here

> /Users/jenkins/opt/jenkins-swarm-client/workspace/enger-transport-mobile-sdk_PR-88/transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt:6:1: Lint error > [indent] Unexpected indentation (8) (should be 4)

File: transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/Log.kt [hunk 2]
```diff
@@ -1,18 +1,29 @@
 package com.genesys.cloud.messenger.transport.util.logs
 
+import co.touchlab.kermit.Severity
 import io.ktor.client.features.logging.Logger
 
-internal expect class Log(enableLogs: Boolean, tag: String) {
+internal class Log(private val enableLogs: Boolean, tag: String) {
 
-    val ktorLogger: Logger
+    val kermit = co.touchlab.kermit.Logger.withTag(tag)
```
Comment (crumpf) [inline transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/Log.kt:8]: `kermit` property can probably be `private`?

Comment (bjdupuis) [reply transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/Log.kt:8]: It's referenced by the `OkHttpLogger`.

File: transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt (hunk 1 above)
Comment (bjdupuis) [reply transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt]: Fixed. I thought I'd run the linter against it. I _had_... then ignored the error. 😬

**Comment by AfanasievAnton** [comment]
Jenkins had a successful build.

**Review by AfanasievAnton — APPROVED** [review]

**Review by crumpf — APPROVED** [review]

**Pull request merged**